packages/use-promise/package.json

```diff
@@ -13,6 +13,10 @@
     "build": "tsc",
     "dev": "tsc --watch"
   },
-  "dependencies": {},
-  "devDependencies": {}
+  "dependencies": {
```

Comment:
I would prefer you'd approve[ my PR](https://github.com/thunderstore-io/thunderstore-ui/pull/997) instead, given that this is a separate issue, my PR has a bit more content, and I asked first :)